x-pack/plugins/ml/public/settings/calendar/edit/_edit.scss

```diff
@@ -0,0 +1,14 @@
+.ml-calendar-form {
+  .ml-calendar-form-content {
+    max-width: 1100px;
+    width: 100%;
+    margin-top: $euiSize;
+    margin-bottom: $euiSize;
+
+    .ml-calendar-combo-box {
+        .euiComboBox__inputWrap--fullWidth {
```

Comment:
Haven't loaded this up to see how you're using it, but remember that `EuiComboBox` has a `fullWidth` prop. Hopefuly that negates your need for the overwrite.

![image](https://user-images.githubusercontent.com/324519/49611196-a7f22200-f955-11e8-80ef-f3bc96a74082.png)